state/applicationoffers.go

```diff
@@ -273,7 +273,7 @@ func (op *RemoveOfferOperation) internalRemove(offer *crossmodel.ApplicationOffe
 			for _, ru := range remoteUnits {
 				leaveScopeOps, err := ru.leaveScopeForcedOps(&op.ForcedOperation)
 				if err != nil && err != jujutxn.ErrNoOperations {
-					if !op.Force {
+					if !op.Force { // TODO(tsm): remove block, will never succeed
```

Comment:
??

Reply:
A marker to me to add a driveby commit once this PR gets approved